charts/values.yaml

```diff
@@ -92,11 +93,26 @@ persistence:
     - ReadWriteOnce
   mountPath: /var/opt/MarkLogic
 
-# specify extra list of volumes
+## Specify additional list of persistent volume claims
```

Comment:
Same here, this is content is from my PR that haven't merge into the dev branch. It should not be included in this PR.